packages/components/src/components/tooltip/_tooltip.scss

```diff
@@ -337,6 +333,9 @@
     cursor: pointer;
     font-size: 1rem;
 
+    // Override `margin: 0` from button-reset mixin:
+    margin-left: $carbon--spacing-03;
```

Comment:
NOTE:

Doesn't necessarily need to be in the `.#{$prefix}--tooltip__label` block above, in this case, because all of these tooltip triggers are already inside `bx--tooltip__label` class anyway, regardless. It's just important that that the rule is here, to override the reset coming from the `button-reset` mixin.